carma-messenger-ui/website/scripts/carmaVersion.php

```diff
@@ -0,0 +1,6 @@
+<?php
+echo shell_exec ("docker container inspect -f '{{ index .Config.Labels \"org.label-schema.version\"}}'  carma-config"); 
```

Comment:
I think this needs to say carma-messenger-config

Reply:
The changes are made and see commit ab3f58c